index.html

```diff
@@ -46,6 +46,312 @@ <h1 id="primary" class="flg">
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">css</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">JavScript</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">html</h5>
+                        </div>
+                    </div>
+                    <div class="see-project">
+                        <button class="btn" type="button">
+                            <span class="btn-text flg">See Project</span>
+                        </button>
+                    </div>
+                </div>
+                <!--Image placeholder goes here-->
+                <div class="img-holder"></div>
+            </div>
+
+            <!--Card Six-->
+            <div class="card flg">
+                <div class="card-text">
+                    <h2 class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </h2>
+                    <div class="tags1 flg">
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">Ruby on rails</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">css</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">JavScript</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">html</h5>
+                        </div>
+                    </div>
+                    <div class="see-project">
+                        <button class="btn" type="button">
+                            <span class="btn-text flg">See Project</span>
+                        </button>
+                    </div>
+                </div>
+                <!--Image placeholder goes here-->
+                <div class="img-holder"></div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About me section-->
+    <section id="about-me">
+        <!-- About me text -->
+        <div id="about-me-text" class="flg">
+            <h1 class="primary flg">About me</h1>
+            <p class="support flg">
+                I can help you build a product , feature or website Look through some of 
+                my work and experience! If you like what you see and have a 
+                project you need coded, don’t hestiate to contact me.
+            </p>
+            <button class="btn-resume" type="button">
+                <span class="btn-text flg">Get My Resume</span>
+            </button>    
+        </div>
+
+        <!--About me bacground images-->
+        <div id="background-image2"><!--Image inserted through css--></div>
+        <div id="background-image3"><!--Image inserted through css--></div>
+
```

Comment:
- [ ] Kindly you should not use different elements to make a background image instead you can get the background image from Figma ad use it as A background for your `about` section in CSS selector to make your project more professional and not get distorted in some instances.

Reply:
yeah, that's my first attempt. but the background image Figma provided for the `about` section is not containing the two SVG on it. please check the template to confirm it.
![illustration-about-me-mobile](https://user-images.githubusercontent.com/106544844/185626340-6755f872-6d8d-4797-a152-6402e8c8ccc9.svg)
![illustration1-about-me-mobile](https://user-images.githubusercontent.com/106544844/185626388-f4ae633a-426d-4b92-baba-75fd06721611.svg)

These are how the background images come. please confirm it.